src/Microsoft.PowerShell.Commands.Utility/resources/WebCmdletStrings.resx

```diff
@@ -138,6 +138,12 @@
   <data name="BodyConflict" xml:space="preserve">
     <value>The cmdlet cannot run because the following conflicting parameters are specified: Body and InFile. Specify either Body or Infile, then retry.  </value>
   </data>
+  <data name="BodyFormConflict" xml:space="preserve">
+    <value>The cmdlet cannot run because the following conflicting parameters are specified: Body and Form. Specify either Body or Form, then retry.  </value>
```

Comment:
I see we use the file pattern but the messages can be shorter - "Conflicting parameters are specified: Body and Form."

Maybe fix this in the PR or new one?

Reply:
There is an existing issue #5140 can you comment there?

Reply:
Done.